deployments/helm-chart/templates/controller-service.yaml

```diff
@@ -29,6 +32,9 @@ spec:
   {{- end }}
 {{- end }}
   type: {{ .Values.controller.service.type }}
+  {{- if .Values.controller.service.ipFamilyPolicy }}
```

Comment:
This field is available since k8s 1.20 so we need a check for the version, something like
```
if semverCompare ">=1.20.0-0" .Capabilities.KubeVersion.Version
```
to avoid problems with k8s < 1.20

Reply:
Does the most recent change resolve the concern?

Reply:
Sorry, I was just checking again the availability of the feature and I think it would be best to have the check for `1.21`, when it reached beta